src/tests/calculate.test.js

```diff
@@ -0,0 +1,8 @@
+import calculate from '../logic/calculate';
+
+test('calculate function adds two numbers', () => {
+  const result = calculate({ total: '2', next: '3', operation: '+' }, '=');
+  expect(result.total).toBe('5');
+});
+
+// Write more tests for the calculate function based on your logic
```

Comment:
Nice work so far. However, as part of the project requirements, kindly make sure that all functionalities and scenarios of the calculate.js modules are covered using jest. :wink: